pyglossary/plugins/stardict.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, glos, filename):
         else:
             self.fileBasePath = filename
         self.fileBasePath = os.path.realpath(self.fileBasePath)
-    
```

Comment:
these blank removals are something automatic from my text editor.  it wasn't my primary intension.